packages/core/src/scss/components/bigcommerce/forms-ccFields/_forms-ccFields.scss

```diff
@@ -99,3 +99,21 @@
 .form-field--setAsDefaultInstrument {
     order: 7;
 }
+
+#paypalcommercecreditcards-ccNumber,
+#paypalcommercecreditcards-ccExpiry,
+#paypalcommercecreditcards-ccName,
+#paypalcommercecreditcards-ccCvv,
+#bigcommerce_payments_creditcards-ccNumber,
+#bigcommerce_payments_creditcards-ccExpiry,
+#bigcommerce_payments_creditcards-ccName,
+#bigcommerce_payments_creditcards-ccCvv {
+  height: 45px;
+  overflow: hidden;
+
+  & > div {
+    top: -5px;
+    left: -5px;
+    width: calc(100% + 10px);
+  }
```

Comment:
The inner PayPal block has additional container with padding, so in this case it is not possible to fully fit the element border by border into our container without such manipulations.

Reply:
Can we use `spacing` utility here please.